network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/tito_widget_block.html

```diff
@@ -0,0 +1,24 @@
+{% extends "./base_streamfield_block.html" %}
+
+{% block block_content %}
+    <script src='https://js.tito.io/v2' async></script>
+
+    {% if value.redirect_page %}
+        <script>
+        // This copes with the javascript being loaded
+        // asycronously (recommended)
+        window.tito =
+            window.tito ||
+            function() {
+            (tito.q = tito.q || []).push(arguments);
```

Comment:
What is this line supposed to do? Because if `window.tito` is undefined, any attempt to access `tito.q` in global scope is going to throw a `ReferenceError: tito is not defined` error.